### PR TITLE
print error value

### PR DESCRIPTION
**Pull request by matthiasdiener**

### CODE REVIEW

**Comment by MTCam** [comment]
Partially addresses https://github.com/annabellej/mirgecompare/issues/21

File: bin/mirgecompare.py [hunk 1]
```diff
@@ -46,7 +46,7 @@ def compare_files_vtu(first_file, second_file, file_type, tolerance = 1e-12):
         # verify individual values w/in given tolerance
         for j in range(arr1.GetSize()):
             if abs(arr1.GetValue(j) - arr2.GetValue(j)) > tolerance: 
-                print("Tolerance:", tolerance)
+                print("Tolerance:", tolerance, "Error:", abs(arr1.GetValue(j) - arr2.GetValue(j)))
                 raise ValueError("Fidelity test failed: Mismatched data array values with given tolerance")
```
Comment (MTCam) [inline bin/mirgecompare.py:50]: This should probably compare them all and record the max difference, then raise if the tolerance was violated.  That way we can see what the max error was.

Comment (MTCam) [inline bin/mirgecompare.py:50]:
```suggestion
        field_errs = [abs(arr1.GetValue(j) - arr2.GetValue(j)) for j in range(arr1.GetSize())]
        max_err = max(field_errs)
        if max_err > tolerance: 
                raise ValueError(f"Fidelity test failed: Mismatched data array values with (tol,err) = ({tolerance}, {max_err})")
```

Would this work?

**Comment by matthiasdiener** [comment]
Done in #710.

**Pull request closed without merging**